examples/nextjs-14-app-dir-rl/middleware.ts

```diff
@@ -18,4 +18,4 @@ const aj = arcjet({
   ],
 });
 
-export default createMiddleware(aj);
\ No newline at end of file
+export default createMiddleware(aj);
```

Comment:
Something strange is happening with the github diff ui. If you view the file in the branch it doesn't have the extra newline.

Reply:
Taken care of